lib/AppInfo/Application.php

```diff
@@ -10,6 +10,8 @@
 
 use OCA\Circles\Events\AddingCircleMemberEvent;
 use OCA\Circles\Events\CircleDestroyedEvent;
+use OCA\Circles\Events\CircleEditedEvent;
```

Comment:
teams/circles edited before that patch don't seem to be updated afterwards? Is it critical, or shoudl we somehow trigger migration step to re-check existing mentions and participants?

Reply:
I don't plan to fix that. Renames are not that often. It can be fixed by renaming twice